pybleau/app/ui/tests/test_dataframe_analyzer_model_view.py

```diff
@@ -197,6 +200,28 @@ def test_apply_filter_button_recomputes_filtered_df_correctly(self):
         view.apply_filter_button = True
         assert_frame_equal(view.model.filtered_df, expected_df)
 
+    @patch.object(FilterExpressionEditorView, "edit_traits")
+    def test_edit_filter_button_recomputes_filtered_df_correctly(self, edit):
+        self.analyzer.filter_auto_apply = False
+        view = DataFrameAnalyzerView(model=self.analyzer, include_plotter=True)
+        view.model.filter_exp = "a != 3"
+        expected_df = DataFrame({"a": [1, 2, 4, 5], "b": [10, 15, 15, 10]},
+                                index=[0, 1, 3, 4])
```

Comment:
Could you refactor this DF to the top of the file and use it here and in the test above?

Reply:
This dataframe is an expected result of a specific filter. Doesn't seem like it should be something that every other function has access to. But I can move it if you think it is necessary.